lib/slack/web/api/templates/method_spec.erb

```diff
@@ -12,7 +12,7 @@ RSpec.describe Slack::Web::Api::Endpoints::<%= group.gsub(".", "_").camelize %>
   context '<%= group %>_<%= name %>' do
 <% required_params.each do |arg_name, arg_v| %>
     it 'requires <%= arg_name %>' do
-<% params_except_required = required_params.reject{ |name, _| name == arg_name  }.map{|var, opts| "#{var}: '#{opts['example'].gsub("'", '"')}'"}.join(', ') %>
+<% params_except_required = required_params.reject{ |name, _| name == arg_name  }.map{ |var, opts| "#{var}: %q[#{opts['example']}]"}.join(', ') %>
```

Comment:
Hmm the docs seem to suggest that interpolation needs the capital %Q, does this work?

Reply:
But we don't want to interpolate here, do we? That string is a copy-paste of an example.

Reply:
Ah ok my bad - brain having some trouble reconciling that the %q is part of the template >_<